runner.sh

```diff
@@ -57,7 +56,11 @@ doChecks=true
 doRun=true
 autofix=false
 failfast=false
-pattern="-and -name '*' -and ! -wholename '*federated_learning*' -and ! -wholename '*unetr_btcv*' -and ! -wholename '*profiling_camelyon*' -and ! -wholename '*profiling_train_base_nvtx*'"
+pattern="-and -name '*' -and ! -wholename '*federated_learning*'\
+ -and ! -wholename '*unetr_btcv*'\
+ -and ! -wholename '*profiling_camelyon*'\
+ -and ! -wholename '*profiling_train_base_nvtx*'\
+ -and ! -wholename '*benchmark_global_mutual_information*'"
```

Comment:
Hi @wyli ,

Is there any nice way to maintain this `runner.sh` file?
I found that we always missed the regex in this file when adding a new notebook..
What do you think?

Thanks in advance.

Reply:
Hi @Nic-Ma, It's not that we forgot to update the file (I have included it in the original PR https://github.com/Project-MONAI/tutorials/pull/438). The root cause is that `pip install -q antspyx` doesn't work on Blossom, but this is not an important issue as we have the core codebase tests to cover it.

Reply:
I see, thanks.